src/Makefile

```diff
@@ -219,13 +219,13 @@ PARSER_NAME     = read_ctrl
 PRINTER_NAME    = print_ctrl
 PARSER_SOURCE   = $(ORAC_COMMON)/common_constants.F90 \
                   $(ORAC_COMMON)/constants_cloud_typing_pavolonis.F90 \
-                  $(ORAC_COMMON)/orac_indexing.F90 ORACConstants.F90 Ctrl.F90
+                  $(ORAC_COMMON)/orac_indexing.F90 orac_constants.F90 ctrl.F90
 USE_STATEMENTS  = "use Ctrl_m; use ORAC_constants_m"
```

Comment:
Shouldn't `use Ctrl_m; use ORAC_constants_m` be converted to `use ctrl_m; use orac_constants_m` ? Fortran isn't case sensitive but for the sake of completeness shouldn't it be consistent with the rest of the filenames ?

Reply:
As you say, Fortran isn't case sensitive so I didn't feel the need to fix this. However, since I'm in the file anyway...